src/autocomplete/popup.js

```diff
@@ -52,6 +52,8 @@ class AcePopup {
         // Set aria attributes for the popup
         popup.renderer.container.setAttribute("role", "listbox");
         popup.renderer.container.setAttribute("aria-label", nls("Autocomplete suggestions"));
+        popup.renderer.container.setAttribute("aria-busy", "true");
```

Comment:
you mentioned `aria-busy` is required due to required children rule described here https://dequeuniversity.com/rules/axe/4.5/aria-required-children?application=axeAPI
which element and with which role requires `aria-busy` to be set for it's child/parent/sibling? I don't see any explicit relations in [aria-busy documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy)

Reply:
Removed in commit 3